api/server/routes/assistants/actions.js

```
@@ -94,16 +99,26 @@ router.post('/:assistant_id', async (req, res) => {
+    if (!assistant_data) {
+      assistantUpdateData.user = req.user.id;
+    }
     promises.push(
       updateAssistantDoc(
         { assistant_id },
-        {
-          actions,
-          user: req.user.id,
-        },
-      ),
+         assistantUpdateData,
```

ESLint

[error] Expected indentation of 8 spaces but found 9.

(indent)